docs/advanced-usage/containers/index.md

```diff
@@ -0,0 +1,5 @@
+# Container technologies
```

Comment:
Usually the index page titles correspond to the section titles, which would call for `Containers`.